force-app/test/mobile/login.spec.ts

```diff
@@ -0,0 +1,34 @@
+import ChooseConn from 'salesforceapp-pageobjects/pageObjects/authentication/chooseConn';
+import Eula from 'salesforceapp-pageobjects/pageObjects/authentication/eula';
+import Login from 'salesforce-pageobjects/helpers/pageObjects/login';
+import LoginNavBar from 'salesforceapp-pageobjects/pageObjects/authentication/loginNavBar';
+import LoginNavBarOptions from 'salesforceapp-pageobjects/pageObjects/authentication/loginNavBarOptions';
+import loaderConfig from 'salesforceapp-pageobjects/mobileSF.config.json';
+
+describe('Test Salesforce App', () => {
+    it('testLogin', async () => {
+        utam.setLoaderConfig(loaderConfig);
```

Comment:
nit: IIRC you can set pass the config specifier directly to `setLoaderConfig` but the way you did is also totally fine